web/themes/custom/novel/templates/layout/node--branch--list-teaser.html.twig

```diff
@@ -0,0 +1,9 @@
+{% include '@novel/components/list-teaser.html.twig'
+  with {
+  attributes: attributes,
+  url: url,
+  image: content.field_main_media.0,
+  baseIconPath: baseIconPath,
+  title: label,
+  description: content.field_address|render|striptags('p')
```

Comment:
Please consider if we can do better that a `striptags` filter.

I think it should be possible to control the rendering of the field using a view mode.

Reply:
As I see it this is still relevant.